man/mkdocs/mkdocs.yml

```diff
@@ -44,6 +44,31 @@ theme:
 # Customization
 extra:
   homepage: ./index.html
+  social:
+    - icon: material/web
+      link: https://grass.osgeo.org
+    - icon: fontawesome/brands/github
+      link: https://github.com/OSGeo/grass
+    - icon: fontawesome/brands/docker
+      link: https://hub.docker.com/r/osgeo/grass-gis
+    - icon: simple/opencollective
+      link: https://opencollective.com/grass
+    - icon: fontawesome/brands/gitter
+      link: https://gitter.im/grassgis/community
+    - icon: fontawesome/brands/mastodon
+      link: https://mastodon.osm-hr.org/@grass
```

Comment:
What is this URL?

I think the correct one is https://fosstodon.org/@grassgis/